dotlottie-rs/src/dotlottie_player.rs

```diff
@@ -265,6 +269,19 @@ impl DotLottieRuntime {
         self.dotlottie_manager.manifest()
     }
 
+    pub fn resolution(&self) -> (u32, u32) {
+        (self.width, self.height)
+    }
+
+    pub fn get_state_machine(&self, state_machine_id: &str) -> Option<String> {
+        let machine = self.dotlottie_manager.get_state_machine(state_machine_id);
+
+        match machine {
+            Ok(machine) => return Some(machine),
+            Err(_) => return None,
+        }
+    }
```

Comment:
[suggestion]
you can convert the Result to Option using .ok() method
```suggestion
   pub fn get_state_machine(&self, state_machine_id: &str) -> Option<String> {
        self.dotlottie_manager.get_state_machine(state_machine_id).ok()
   }
```